examples/integrations/oci/vault-cdi/src/main/java/io/helidon/examples/integrations/oci/vault/cdi/VaultResource.java

```diff
@@ -18,51 +18,68 @@
 
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
-import java.util.Optional;
+import java.util.Date;
 
 import javax.inject.Inject;
-import javax.inject.Named;
 import javax.ws.rs.DELETE;
 import javax.ws.rs.GET;
-import javax.ws.rs.NotFoundException;
+import javax.ws.rs.InternalServerErrorException;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 
 import io.helidon.common.Base64Value;
-import io.helidon.integrations.oci.vault.CreateSecret;
-import io.helidon.integrations.oci.vault.Decrypt;
-import io.helidon.integrations.oci.vault.DeleteSecret;
-import io.helidon.integrations.oci.vault.Encrypt;
-import io.helidon.integrations.oci.vault.GetSecretBundle;
-import io.helidon.integrations.oci.vault.OciVault;
-import io.helidon.integrations.oci.vault.Sign;
-import io.helidon.integrations.oci.vault.Verify;
 
+import com.oracle.bmc.keymanagement.KmsCryptoClient;
+import com.oracle.bmc.keymanagement.model.DecryptDataDetails;
+import com.oracle.bmc.keymanagement.model.EncryptDataDetails;
+import com.oracle.bmc.keymanagement.model.SignDataDetails;
+import com.oracle.bmc.keymanagement.model.VerifyDataDetails;
+import com.oracle.bmc.keymanagement.requests.DecryptRequest;
+import com.oracle.bmc.keymanagement.requests.EncryptRequest;
+import com.oracle.bmc.keymanagement.requests.SignRequest;
+import com.oracle.bmc.keymanagement.requests.VerifyRequest;
+import com.oracle.bmc.secrets.SecretsClient;
+import com.oracle.bmc.secrets.model.Base64SecretBundleContentDetails;
+import com.oracle.bmc.secrets.model.SecretBundleContentDetails;
+import com.oracle.bmc.secrets.requests.GetSecretBundleRequest;
+import com.oracle.bmc.vault.VaultsClient;
+import com.oracle.bmc.vault.model.Base64SecretContentDetails;
+import com.oracle.bmc.vault.model.CreateSecretDetails;
+import com.oracle.bmc.vault.model.ScheduleSecretDeletionDetails;
+import com.oracle.bmc.vault.model.SecretContentDetails;
+import com.oracle.bmc.vault.requests.CreateSecretRequest;
+import com.oracle.bmc.vault.requests.ScheduleSecretDeletionRequest;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 
 /**
  * JAX-RS resource - REST API of the example.
  */
 @Path("/vault")
 public class VaultResource {
-    private final OciVault vault;
+    private final SecretsClient secrets;
```

Comment:
I won't flag these as changes that need to be made, but it would be better if anything that has "`Client`" appended to it could be changed to just the interface.

Reply:
(Among other things this permits simple `@Alternative`-based mocking in unit tests.  I know this is an example but code gets copied and pasted.)